foxy/distribution.yaml

```diff
@@ -1735,7 +1735,7 @@ repositories:
       url: https://github.com/ros2-gbp/rosbag2_bag_v2-release.git
       version: 0.0.10-1
     source:
-      test_pull_requests: true
+      test_pull_requests: false
```

Comment:
This line should be removed since `false` is the default. Additionally it needs a line `test_commits: false` to get rid of the `dev` job.

Reply:
done in https://github.com/ros/rosdistro/pull/25505/commits/5988f98a416fb833e598401a4019eb79f687955f